From 49d19f1c922da527df10b9f1d7b758ca36246d0e Mon Sep 17 00:00:00 2001
From: "Liu,Dalei" <daleix.liu@intel.com>
Date: Fri, 20 Nov 2015 14:33:41 +0800
Subject: [PATCH 20/28] atomisp: Increase ISP frequency of 1080P video

Increase ISP frequency for 30fps 1080P video recording

Change-Id: I338c3b31389308a774915109e6c28ab04c99d6ac
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9279
Original-Tracked-On: #H1504085120
Signed-off-by: Liu,Dalei <daleix.liu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/85867
Approver: Bin B Han <bin.b.han@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: Dessert M COHO (SC)
Build: Dessert M BXT RVP (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Signed-off-by: Yang Fan F <fan.f.yang@intel.com>
Reviewed-on: https://android.intel.com:443/442543
---
 .../drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h
index f7cedcd..2ba88da 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h
@@ -145,7 +145,7 @@ static const struct atomisp_freq_scaling_rule dfs_rules_merr_117a[] = {
 		.width = 1080,
 		.height = 1920,
 		.fps = 30,
-		.isp_freq = ISP_FREQ_266MHZ,
+		.isp_freq = ISP_FREQ_400MHZ,
 		.run_mode = ATOMISP_RUN_MODE_VIDEO,
 	},
 	{
-- 
1.9.1

